### ISSUE TITLE
Alternative Method for Windows?

### ISSUE THREAD

**Issue by ConHNation** [ccorcos/hack#10]
Hey

Most of my friends use Windows, and I'm fairly new to Node.js. Wasn't able to find an alternative way to use this on a Windows PC. Since most of my friends use Windows as their OS, is there any way I could use this on them?

**Comment by ccorcos** [comment]
Yeah, would have to use a windows cron instead of crontab for mac:
https://stackoverflow.com/questions/7195503/setting-up-a-cron-job-in-windows

**Comment by ccorcos** [comment]
My package only works for mac currently thoguh
